src/UpsilonGC/inc/UpsilonGCHandleManager.h

```diff
@@ -8,9 +11,13 @@ class UpsilonGCHandleManager : public IGCHandleManager
 	// Inherited via IGCHandleManager
     virtual bool Initialize() override;
     virtual void Shutdown() override;
-    virtual void * GetHandleContext(OBJECTHANDLE handle) override;
+    virtual void * GetHandleContext(OBJECTHANDLE handle);
     virtual IGCHandleStore * GetGlobalHandleStore() override;
+#ifdef __linux__
+    virtual IGCHandleStore * CreateHandleStore() override;
+#else
     virtual IGCHandleStore * CreateHandleStore(void * context) override;
+#endif
```

Comment:
Is this because you need to support newer runtime in your case? If so, I would like to avoid #ifdef __linux__ and just bump up the supported GC API version by the Upsilon GC (no matter the operating system).